python/pyproject_revng_internal.toml

```diff
@@ -0,0 +1,46 @@
+# This file is distributed under the MIT License. See LICENSE.md for details.
```

Comment:
What's the `pyproject_` prefix for?

Reply:
The "correct" name of the file is `pyproject.toml`. Since we need the `revng` and `revng` internal version, those have been used as a postfix.

Reply:
That's not the structure that python expects. for both packages the structure before running `python -m build` should be:
```
drwxrwxr-x python
-rw-r--r-- pyproject.toml
```

Reply:
Ah OK, the conventions are important for the CMake magic to happen, correct?
CMake looks up `pyproject_$X.toml` where `$X` is the (normalized) name of the package?

Reply:
Correct on both